ts/pages/api.tsx

```diff
@@ -0,0 +1,830 @@
+import { BigNumber } from '@0x/utils';
+import React, { useState } from 'react';
+import CopyToClipboard from 'react-copy-to-clipboard';
+import SyntaxHighlighter from 'react-syntax-highlighter';
+import dark from 'react-syntax-highlighter/dist/cjs/styles/hljs/vs2015';
+import styled from 'styled-components';
+
+import { AnimatedChatIcon } from 'ts/components/animatedChatIcon';
+import { AnimatedCompassIcon } from 'ts/components/animatedCompassIcon';
+import { AnimationLoader } from 'ts/components/animations/animation_loader';
+import { BlockIconLink } from 'ts/components/blockIconLink';
+import { Button } from 'ts/components/button';
+import { DocumentTitle } from 'ts/components/document_title';
+import { Hero } from 'ts/components/hero';
+import { Section } from 'ts/components/newLayout';
+import { SiteWrap } from 'ts/components/siteWrap';
+
+import { colors } from 'ts/utils/colors';
+import { constants } from 'ts/utils/constants';
+import { documentConstants } from 'ts/utils/document_meta_constants';
+import { Translate } from 'ts/utils/translate';
+
+import { WebsitePaths } from 'ts/types';
+
+export interface ApiPageProps {
+    location: Location;
+    translate: Translate;
+}
+
+const ZeroExApi: React.FC<ApiPageProps> = () => {
+    const [isCopied, setIsCopied] = React.useState<boolean>(false);
+    const copyButtonText = isCopied ? 'Copied!' : 'Copy to clipboard';
+
+    const handleCopyClick = () => {
+        setIsCopied(true);
+        setTimeout(() => setIsCopied(false), 1000);
+    };
+
+    const [quote, setQuote] = useState<any | undefined>();
+
+    const takerAmount = new BigNumber(100000000); // 100$ in usdc (only 8 digits?)
+    const formattedTakerAmount = takerAmount // 100$ in usdc (only 8 digits?)
+        .dividedToIntegerBy(1)
+        .toString();
+    const quoteEndpoint = `https://api.0x.org/swap/v0/quote?sellAmount=${formattedTakerAmount}&buyToken=DAI&sellToken=USDC`;
```

Comment:
Not sure if the hard code was intentional but we have `utils.getAPIBaseUrl(networkId)` that will return an environment specific API base url.

Reply:
Fixed! Thanks